web/src/components/molecules/Schema/FieldModal/FieldCreationModal/index.tsx

```diff
@@ -124,7 +131,21 @@ const FieldCreationModal: React.FC<Props> = ({ open, onClose, onSubmit, selected
             <Form.Item
               name="key"
               label="Field Key"
-              rules={[{ required: true, message: "Please input the key of field!" }]}>
+              rules={[
+                { required: true, message: "Please input the key of the field!" },
+                {
+                  message: "Key is not valid",
+                  validator: async (_, value) => {
+                    if (!/^[a-zA-Z0-9_-]{5,32}$/.test(value)) return Promise.reject();
```

Comment:
Can you put all regex tests in a utils file? 
Also, are these regexs arbitrary or are they part of the planned CMS architecture?

Reply:
@KaWaite , I got this regex from @yk-eukarya , it was defined in requirements to have a URL-friendly key with a length between 5 and 32
Sure, I will move it to a utils file

Reply:
@KaWaite 
https://github.com/reearth/reearth-cms/blob/da31fd55cfbccd30f05b2841bf37b97a927c6936/web/src/utils/regex.ts#L1-L5
Used like this:
https://github.com/reearth/reearth-cms/blob/da31fd55cfbccd30f05b2841bf37b97a927c6936/web/src/components/molecules/Schema/FieldModal/FieldCreationModal/index.tsx#L144
Please check, and advise if the name of the file and function is good.

Reply:
Good!